EMS/Enquiry/admin.py

```diff
@@ -0,0 +1,4 @@
+from django.contrib import admin
+from .models import Enquiry
```

Comment:
Always use absolute imports and not relative imports
```suggestion
from enquiries.models import Enquiry
```